functions/brinks/src/index.js

```diff
@@ -0,0 +1,69 @@
+import 'dotenv/config';
+import express from 'express';
+import debug from 'debug';
+import * as Sentry from '@sentry/node';
+import responseTime from 'response-time';
+import {
+  decodeMessage,
+  eventProcessedCheck,
+  stripeEventCsCompleted,
+  stripeEventCustCreated,
+  stripeEventEnd,
+  errorHandler,
+} from './middleware';
+
+const dlog = debug('that:api:functions:brinks');
+const api = express();
+
+let version;
+(async () => {
+  let p;
+  try {
+    p = await import('./package.json');
+  } catch {
+    p = await import('../package.json');
+  }
+  version = p.version;
+})();
+const defaultVersion = `brinks@${version}`;
+dlog('setting up');
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
+  environment: process.env.THAT_ENVIRONMENT,
+  release: process.env.SENTRY_VERSION || defaultVersion,
+  debug: process.env.NODE_ENV === 'development',
+});
+Sentry.configureScope(scope => {
+  scope.setTags({
+    thatApp: 'brinks',
+    subSystem: 'checkout',
+  });
+});
+
+// eslint-disable-next-line no-unused-vars
+function dumpBody(req, res, next) {
+  dlog('dump body called');
+  console.log('method::', req.method);
+  console.log('body::', req.body);
+  // res.status(200).end();
+  next();
+}
+
+export const handler = api
```

Comment:
my only real question here is should we be setting firebase here to be reused downstream.

Reply:
Yeah I suppose. Not sure how that would look. I assume it attached to the req object.  I also wonder if it means it is still created in the global space which is said to be a best practice for google functions. https://cloud.google.com/functions/docs/bestpractices/tips#use_global_variables_to_reuse_objects_in_future_invocations

Reply:
There are some comments around this in other areas of this review

Reply:
Refactored in brinks: ac099c2